src/main/cpp/subsystems/MAXSwerveModule.cpp

```diff
@@ -170,3 +170,32 @@ void MAXSwerveModule::simUpdateDrivePosition(
 }
 
 void MAXSwerveModule::ResetEncoders() { m_drivingEncoder.SetPosition(0); }
+
+void MAXSwerveModule::SetMotorVoltage(MotorType type, units::volt_t voltage) {
+  if (type == MotorType::DriveMotor) {
+    m_drivingSparkMax.SetVoltage(voltage);
+    return;
+  }
+  m_turningSparkMax.SetVoltage(voltage);
+}
+
+double MAXSwerveModule::Get(MotorType type) {
```

Comment:
Is this returning the % of max voltage that was last set? Need a better name than `Get`